sympy/parsing/latex/lark/transformer.py

```diff
@@ -42,10 +42,16 @@ class TransformToSymPyExpr(Transformer):
 
         Note that the option must be set to ``True`` for the default parser to work.
     """
+    evaluate = None
 
     SYMBOL = sympy.Symbol
     DIGIT = sympy.core.numbers.Integer
 
+    def transform_tree(self, tree, evaluate):
+        TransformToSymPyExpr.evaluate = evaluate
```

Comment:
This looks like it is setting a global variable. The mechanism should not use a global variable like this which would e.g. not work across multiple threads.

Reply:
Why not define `__init__` with with an argument `evaluate`